include/engine/routing_algorithms/routing_base_mld.hpp

```diff
@@ -326,27 +331,18 @@ inline void search(SearchEngineData<Algorithm> &engine_working_data,
                    const PhantomNodes &phantom_nodes,
                    const EdgeWeight weight_upper_bound = INVALID_EDGE_WEIGHT)
 {
-    NodeID source_node, target_node;
-    std::vector<EdgeID> unpacked_edges;
-    std::tie(weight, source_node, target_node, unpacked_edges) = mld::search(engine_working_data,
-                                                                             facade,
-                                                                             forward_heap,
-                                                                             reverse_heap,
-                                                                             force_loop_forward,
-                                                                             force_loop_reverse,
-                                                                             weight_upper_bound,
-                                                                             phantom_nodes);
-
-    if (weight != INVALID_EDGE_WEIGHT)
-    {
-        packed_leg.push_back(source_node);
-        std::transform(unpacked_edges.begin(),
-                       unpacked_edges.end(),
-                       std::back_inserter(packed_leg),
-                       [&facade](const auto edge) { return facade.GetTarget(edge); });
-    }
+    // TODO: change search calling interface to use unpacked_edges result
+    std::tie(weight, packed_leg, std::ignore) = mld::search(engine_working_data,
```

Comment:
The name `packed_leg` is actually wrong here, right? The search returns `unpacked_nodes` at this position.